modules/microshift-building-apps-rpms.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// microshift_running_applications/embedding-apps-tutorial.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-building-apps-rpms_{context}"]
+= Building the RPM package for the application manifests
+
+To building your own RPMs, you must create a `.spec` file that adds the application manifests to the RPM package. The following is an example procedure. As long as the application RPMs and other elements needed for image building are accessible to Image Builder, you can use the method that you prefer.
```

Comment:
You can use "spec file," no backticks. See the glossary: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#spec